src/Rule/CompareToHandler.php

```diff
@@ -39,13 +39,17 @@ public function validate(mixed $value, object $rule, ?ValidationContext $context
         }
 
         $result = new Result();
+        $compareAttribute = $rule->getCompareAttribute() ?? $context?->getAttribute() . '_repeat';
```

Comment:
What is `_repeat`?

Reply:
Added phpdocs, the logic is ported from Yii2.

Reply:
I still don't see a logic of it. If you want to compare `password` and `password_repeat`, then specify values.
```php
class SomeDTO
{
  #[Compare(attribute: 'passwordRepeat')
  private $password;
  #[Compare(attribute: 'password')
  private $passwordRepeat;
}
```

Reply:
OK. Logic removed.